docs/manage-application.md

```diff
@@ -0,0 +1,53 @@
+---
+id: manage-application
+title: Manage a ReSolve Application
+---
+
+Use the [resolve-module-admin](https://github.com/reimagined/resolve/tree/master/packages/modules/resolve-module-admin#readme) CLI tool to manage a reSolve application's sagas and read models. It provides the following functionality:
```

Comment:
`It provides the following functionality:`

I'd like to update this without using "provides" if possible. Can we say "includes" instead?

Reply:
Yes, changed to "includes"

Reply:
Thank you.